pkg/shell/indexes.js

```diff
@@ -143,6 +143,18 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
     });
 
     function show_disconnected() {
+        if (!ready) {
+            let ca_cert_url = window.sessionStorage.getItem("CACertUrl");
+            $("#early-failure").show();
+            if (window.navigator.userAgent.indexOf("Safari") >= 0 && ca_cert_url) {
```

Comment:
At least my Chrome has this UserAgent by default: `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36`

Can we get more specific in the test, or are we fine to just the message even if it doesn't apply?

Reply:
Supposedly the test could set the user agent explicitly in both cases, to avoid relying on that (really weird) default?

Reply:
Yes, the test is fine, but people with a Chrome browser will see the paragraph intended for people with Safari browsers.

Reply:
Ugh, yes. So in a follow-up we could fine-tune the userAgent check, but we should check what a real Safari has on macOS and iOS.